### PR TITLE
Adjust namespacing

### PR DESCRIPTION
**Pull request by dcsjapan**
- Updates namespacing of all translation keys.
- Namespacing of settings is unchanged.

### CODE REVIEW

**Comment by dcsjapan** [comment]
I think I've got it right this time. 😃

File: js/lib/helpers/tagLabel.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function tagLabel(tag, attrs = {}) {
   return (
     m((link ? 'a' : 'span'), attrs,
       <span className="TagLabel-text">
-        {tag ? tag.name() : app.translator.trans('flarum-tags.lib.deleted_tag_text')}
+        {tag ? tag.name() : app.translator.trans('tags::deleted_tag_text')}
```
Comment (franzliedke) [inline js/lib/helpers/tagLabel.js]: @dcsjapan So `lib` can be removed like `admin` as well?

Just asking for confirmation as this seems to be the only exception...

**Comment by franzliedke** [comment]
Thank you very much! All changes look syntactically correct (just waiting for a reply to that one question). I did not check whether you missed anything, but if so, we'll notice that later anyway. :)

**Comment by issyrocks12** [comment]
Yes they can be removed, same as admin and forum

**Comment by dcsjapan** [comment]
Closed as @tobscure has concluded removal of the scope from key names isn't worth the extra fuss.

**Comment by franzliedke** [comment]
@dcsjapan Did I miss that? Where did he say so?

**Comment by tobscure** [comment]
@franzliedke in a private discussion with @dcsjapan

**Comment by dcsjapan** [comment]
Sorry, I could have phrased that more clearly!

**Pull request closed without merging**